app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt

```diff
@@ -33,12 +34,22 @@ class ConversationViewModel @Inject constructor(
     private val navigationManager: NavigationManager,
     private val getMessages: GetRecentMessagesUseCase,
     private val getConversationDetails: GetConversationDetailsUseCase,
-    private val sendTextMessage: SendTextMessageUseCase
+    private val sendTextMessage: SendTextMessageUseCase,
+    private val deleteMessage: DeleteMessageUseCase
 ) : ViewModel() {
 
     var conversationViewState by mutableStateOf(ConversationViewState())
         private set
 
+    var deleteMessageDialogsState: DeleteMessageState by mutableStateOf(
```

Comment:
I think we can move it into ConversationViewState()

Reply:
Yes, but the structure of the deleteMessageDialogsState is better actually we need to refactor conversationViewState to the same model as this one.

Reply:
Honestly, we should refactor conversationViewState based on this structure and Sealed classes. so I think if we need to move this stuff together would be un needed job with this PR.